ecommerce-product-page-main/styled.css

```diff
@@ -367,45 +333,29 @@ ul.list li a:hover{
 
 
 
-.button1{
-
+.discount{
     font-family: 'Kumbh Sans', sans-serif;
     font-weight: 700;
     font-size: 12px;
     color: hsl(26,100%,55%);
     background-color: hsl(26, 100%, 94%);
-    max-width: 45px;
-    max-height: 30%;
+    flex-shrink: 1;
     margin-top: 20px;
-    border-radius: 5px;
+    border-radius: 5px;  
     border-style: none;
-    
+
 }
 
 .imgbutt{
-    
-    max-width: 20%;
-    max-height: 10%;
-    margin-right: 10px;
-    margin-top: 10px;
-    margin-left: 50px;
-
-    
-
-    
-    
+    width: 20px;
 }
 
-h3{
-
-    width: 100%;
-    max-height: 10%;
-    
-}
 
-.button2{
+.addToCartButton{
     display: flex;
-    flex-direction: row;
+    justify-content: center;

```

Comment:
Was this what you used to get the action that i got with my css in my line 408?

Reply:
no. read up on flex properties